### ISSUE TITLE
ubuntu 22.10

### ISSUE THREAD

**Issue by ninmba** [achaphiv/ppa-fonts#47]
can you build for ubuntu 22.10?
thanks

**Comment by achaphiv** [comment]
Sadly, no.

This repo is pretty dead at this point.

- I relied on other people to provide the infinality patch. I think I've used patches from 4+ different maintainers. They're all dead now. I can't really blame them, since the infinality patch needs to be constantly updated for every freetype version (currently v2.12.1). Historically, even incremental versions required a separate patch.
- I've stopped using ubuntu, because my hardware required newer kernels.

**Comment by greatquux** [comment]
Yeah, I have no hope of these patches ever coming back, but on the bright side, I'm pretty sure some of this work eventually made its way into mainstream Freetype, and with the continuing upgrades to HiDPI displays, font smoothing tech will become less important as the accurate font shapes can be displayed more truly.
